src/net_processing.h

```diff
@@ -27,13 +28,19 @@ static constexpr int64_t HEADERS_DOWNLOAD_TIMEOUT_PER_HEADER = 1000; // 1ms/head
 static constexpr int32_t MAX_OUTBOUND_PEERS_TO_PROTECT_FROM_DISCONNECT = 4;
 /** Timeout for (unprotected) outbound peers to sync to our chainwork, in seconds */
 static constexpr int64_t CHAIN_SYNC_TIMEOUT = 20 * 60; // 20 minutes
+/** How frequently to check for stale tips, in seconds */
+static constexpr int64_t STALE_CHECK_INTERVAL = 10 * 60; // 10 minutes
+/** How frequently to check for extra outbound peers and disconnect, in seconds */
+static constexpr int64_t EXTRA_PEER_CHECK_INTERVAL = 45;
+/** Minimum time an outbound-peer-eviction candidate must be connected for, in order to evict, in seconds */
+static constexpr int64_t MINIMUM_CONNECT_TIME = 30;
```

Comment:
I think we should probably bump this to 60sec, to match the minimum time allowed for the handshake. Or was 30sec a deliberate choice?

That would also give us the handy property that all fSuccessfullyConnected nodes can be disconnected without checking the connection duration.

Reply:
>  disconnected without checking the connection duration.

Huh.. then you'd disconnect someone who completed the handshake fast, without giving them more than a few seconds to transfer blocks.

Reply:
Sorry, I meant that in the context of the check here: https://github.com/bitcoin/bitcoin/pull/11560/commits/21e8b03a353abd0f2f4190194f224ad92d6f5bc8#diff-eff7adeaec73a769788bb78858815c91R3055

Reply:
I dont really think it matters here - these are separate, we're seeking a new useful outbound peer, if it doesn't respond in 30 seconds with a headers message, it probably isnt/wont be useful. Honestly I think we should significantly decrease the headers handshake timeout, there isn't really a scenario where you should need 60 seconds for a few packets back and forth, even in super lossy environments.

Reply:
ping @TheBlueMatt